prac_05/emails.py

```diff
@@ -0,0 +1,15 @@
+email_list = {}
+email = input("Enter email: ")
+while email != "":
+    email_name = email.split('@')
```

Comment:
I think you could save some space by just using "email_name = (email.split("@")[0]).title()"
It formats it into a title format and you don't need to keep referring to the list index.